src/vg/civcraft/mc/citadel/listener/EntityListener.java

```diff
@@ -154,6 +179,271 @@ public void PlayerPromoteEvent(PromotePlayerEvent event){
     			
     		}
     	}
-    }   	
- 
+    }
+    
+    @EventHandler(priority = EventPriority.HIGHEST)
+    public void entityInteration(EntityDamageEvent event){
+    	
```

Comment:
Good catch, missed removing that test code trying to figure out the correct event I needed to hook.